how2meet/ui/pages/urls.py

```diff
@@ -0,0 +1,8 @@
+ROUTE_PREFIX_EVENTS = "/events"
+ROUTE_EVENTS_LIST = "/"
+ROUTE_NEW_EVENT = "/new_event"
+ROUTE_EVENT_HOME = "/home/{event_id}"
```

Comment:
This doesn't matter thaaaaat much but I don't know if I like this url naming scheme. I would probably expect this to be /events/{event_id} and be named something like `ROUTE_EVENT_DETAILS`. Just a slightly more CRUD-y pattern.